_posts/2021-09-23-hacktoberfes.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: Hacktoberfest 2021
+layout: post
+---
+
+Are you ready for Hacktoberfest 2021?  Void Linux is!  This year we're
+excited to be participating for our //// year.  We're looking for
+contributions to assist this year with our out-of-date packages queue,
+and these are all great ways to dip your feet into the world of Linux
+distro package management, and what happens behind the scenes to make
+sure your system is up to date and complete with a wide selection of
+packages.
+
+Updating packages is very easy.  You can select a package from the
+list of [out of date
+packages](http://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt)
+and update it using the tools in the [void-packages
+repo](https://github.com/void-linux/void-packages).  The
+[manual](https://github.com/void-linux/void-packages/blob/master/Manual.md)
+might be of assistance when you are updating packages.
+
+As a general rule, we would prefer Hacktoberfest contributors that do
+not have a previous track record with the Void Linux project steer
+clear of "structural" packages unless you have specific domain
+knowledge that qualifies you to work on high-risk packages.  When
+selecting a package to update, select a package registered to
+`orphan@voidlinux.org` when possible.  These packages are otherwise
+unmaintained, and your contribution will have a bigger impact.  You
+can update packages that are owned by existing developers, but
+understand that conflicts between a maintainer and contributor will be
+resolved at the discretion of Void staff.
```

Comment:
```suggestion
knowledge that qualifies you to work on high-risk packages.  When
selecting a package to update, prefer packages registered to
`orphan@voidlinux.org`.  These packages are otherwise
unmaintained, and your contribution will have a bigger impact.  You
can update packages that have a maintainer assigned, but
understand that conflicts between a maintainer and contributor will be
resolved at the discretion of Void staff.
```
This section could be interpreted as "please don't touch packages which are not orphaned" which is not the intention. The changes I suggested make it a bit better, but the last part still sounds a bit too hostile in my opinion: 

> but understand that conflicts between a maintainer and contributor will be
resolved at the discretion of Void staff.